cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/ControlDependenceGraphPass.kt

```diff
@@ -319,6 +338,9 @@ private fun EvaluationOrder.isConditionalBranch(): Boolean {
         true
     } else
         (this.start is IfStatement ||
+            this.start is ComprehensionExpression ||
```

Comment:
Here we have a similar issue.

Reply:
This piece of code is slightly different and we want to figure out if e.g. the "false" branch of the statement is executed after the "true" branch. For this, we probably still want to make the decision based on the individual nodes.